First case.R

```diff
@@ -75,8 +75,8 @@ deaths_cases_indicators <- merge(deaths_cases_population, indicators, by =c("cou
 # Cleaning up columns
 
 deaths_cases_indicators <- deaths_cases_indicators %>% select(country_code, Cases, Deaths, clean_date, day_since_first_case, 
-                                   pop_2019, casepc, deathpc, cfratio, overall, prev_emergence_pathogens, early_detection,
-                                   rapid_response, robust_health_sector, commitments, risk_environment)
+                                                              pop_2019, casepc, deathpc, cfratio, overall, prev_emergence_pathogens, early_detection,
```

Comment:
It seems like this file wasn't actually changed. Reverting it might resolve the merge conflict

Reply:
This should be reverted back now!